spec/features/merchants/items/index_spec.rb

```diff
@@ -6,6 +6,7 @@
     @merchant_1 = Merchant.create!(name: "Staples")
     @item_1 = @merchant_1.items.create!(name: "stapler", description: "Staples papers together", unit_price: 13)
     @item_2 = @merchant_1.items.create!(name: "paper", description: "construction", unit_price: 29)
+    @enabled_item = @merchant_1.items.create!(name: "pen", description: "red ink", unit_price: 5, status: "Enabled")
```

Comment:
Can we move this out of the before block and into the test you need it for? I think it makes it more readable for this test, but may mess us up in the future.